README.md

```diff
@@ -13,7 +13,7 @@ LORIS (Longitudinal Online Research and Imaging System) is a self-hosted web app
 This Readme covers installation of LORIS version <b>23</b> on <b>Ubuntu</b>.
 ([CentOS Readme also available](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md)).
 
-Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+Please consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).
```

Comment:
`Please consult the [Ubuntu Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md) or [CentOS Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).`

Reply:
This should be compatible with readthedocs if we want the main README to be part of it in the future.

Reply:
Ok I can do it this way. Sorry I was confused about what you meant in previous comments. Thanks :)